tests/test_data.py

```diff
@@ -136,4 +136,8 @@ def test_Kirk19_ingest(db):
     ref = 'Kirk19'
     t = db.query(db.ProperMotions).filter(db.ProperMotions.c.reference == ref).astropy()
     assert len(t) == 22, f'found {len(t)} proper motion entries for {ref}'
-    
+
+    #Test photometry added
```

Comment:
Also add a test for photometry with the "Kirk19" reference.

Reply:
So the only thing I'm a bit confused about is that it looks like there are 9 objects in the csv file with Kirk19 as the spitzer reference, but 16 total in the database now (which means 8 objects, since each is in there twice). Did it miss one? Should I put "assert len(t) == 16" or assert len(t) == 18"?

Reply:
Maybe one of the Kirk19 sources is the first one which we are skipping? 

And i forgot about the other ref...add that as a test too. :)

Reply:
It's the first one which we're skipping! That explains it. I'll add a test for the other source too